code/modules/projectiles/ammunition/ammo_kits.dm

```diff
@@ -50,85 +50,92 @@
+			"5 points: EMPTY carbine standard magazine" = list(5, /obj/item/ammo_magazine/light_rifle_257_short/empty),
+			"10 points: EMPTY carbine extended magazine" = list(10, /obj/item/ammo_magazine/light_rifle_257/scrap/empty),
+			"30 points: EMPTY carbine drum magazine" = list(30, /obj/item/ammo_magazine/light_rifle_257_drum/empty),
+			"20 points: EMPTY carbine linked box" = list(20, /obj/item/ammo_magazine/rifle_75_linked_box/light_rifle_257/empty)
 			),
-		CAL_BALL = list(
-			"17mm ball ammo pile (4 ammo, 1 points)" = list(1, /obj/item/ammo_casing/ball/prespawned),
-			"17mm ball ammo box (30 ammo, 15 points)" = list(15, /obj/item/ammo_magazine/ammobox/ball)
+		CAL_RIFLE = list(
+			"3 points, 10 ammo: Scrap ammo pile" = list(3, /obj/item/ammo_casing/rifle_75/scrap/prespawned),
+			"15 points, 30 ammo: Scrap rifle ammo packet" = list(15, /obj/item/ammo_magazine/ammobox/rifle_75_small/scrap),
+			"8 points, 10 ammo: Rifle ammo pile" = list(8, /obj/item/ammo_casing/rifle_75/prespawned),
+			"26 points, 30 ammo: Rifle ammo packet" = list(26, /obj/item/ammo_magazine/ammobox/rifle_75_small),
+			"5 points: EMPTY Rifle short magazine" = list(5, /obj/item/ammo_magazine/rifle_75_short/scrap/empty),
+			"10 points: EMPTY Rifle extended magazine" = list(10, /obj/item/ammo_magazine/rifle_75/empty),
+			"30 points: EMPTY Rifle drum magazine" = list(30, /obj/item/ammo_magazine/rifle_75_drum/empty),
+			"20 points: EMPTY Rifle linked box" = list(20, /obj/item/ammo_magazine/rifle_75_linked_box/empty)
+			),
+		CAL_HRIFLE = list(
+			"3 points, 10 ammo: Scrap ammo pile" = list(3, /obj/item/ammo_casing/heavy_rifle_408/scrap/prespawned),
+			"15 points, 40 ammo: Scrap ammo box" = list(15, /obj/item/ammo_magazine/ammobox/heavy_rifle_408_small/scrap),
+			"8 points, 10 ammo: ammo pile" = list(8, /obj/item/ammo_casing/heavy_rifle_408/prespawned),
+			"26 points, 40 ammo: ammo packet" = list(26, /obj/item/ammo_magazine/ammobox/heavy_rifle_408_small),
+			"5 points: EMPTY heavy rifle magazine" = list(5, /obj/item/ammo_magazine/heavy_rifle_408/scrap/empty),
+			"30 points: EMPTY heavy rifle drum magazine" = list(30, /obj/item/ammo_magazine/heavy_rifle_408_drum/empty)
 			),
 		CAL_ANTIM = list(
-			"Scrap 14.5mm ammo pile (3 ammo, 5 points)" = list(5, /obj/item/ammo_casing/antim/scrap/prespawned),
-			"Scrap 14.5mm ammo box (5 ammo, 10 points)" = list(10, /obj/item/ammo_magazine/ammobox/antim/scrap),
-			"14.5mm ammo box (3 ammo, 25 points)" = list(20, /obj/item/ammo_casing/antim/prespawned),
-			"14.5mm ammo box (15 ammo, 30 points)" = list(30, /obj/item/ammo_magazine/ammobox/antim_small)
+			"5 points, 3 ammo: Scrap 14.5mm ammo pile" = list(5, /obj/item/ammo_casing/antim/scrap/prespawned),
+			"10 points, 5 ammo: Scrap 14.5mm ammo box" = list(10, /obj/item/ammo_magazine/ammobox/antim/scrap),
+			"10 points, 3 ammo: 14.5mm ammo pile" = list(10, /obj/item/ammo_casing/antim/prespawned),
```

Comment:
way to cheap and shouldnt have real antimatter ammo

Reply:
The real antimatter ammo was already there though, it's still nearly impossible to get 30 points even on the machine, but I can up the prices if you want
This was done thinking on amount of bullets compared to the price of the next bigger caliber before it

Reply:
ive gotton 54 points e-z
but ya apparently i added real for some reason so keep it 30